documentation/4.0/content/release-notes.md

```diff
@@ -65,6 +66,26 @@ draft: false
 
 ### Change log
 
+#### Operator 4.0.5
+
+* Updated the Fluentd integration to make the Elasticsearch credentials optional and to allow the specification of the container command and arguments.
+* Resolved an issue related to growth in the number of WebLogic sessions related to the monitoring exporter and health checks.
+* Resolved [issue #3865](https://github.com/oracle/weblogic-kubernetes-operator/issues/3865) related to decorating the name of the ConfigMap for Fluentd integration ([#3883](https://github.com/oracle/weblogic-kubernetes-operator/pull/3883)).
+* Updated the generation of pods and containers for [security best practices](https://oracle.github.io/weblogic-kubernetes-operator/security/pod-and-container/).
```

Comment:
-> I think that the x-ref should be to /security/**domain-security**/pod-and-container/